### DISCUSSION TITLE
[Bug]: Failed building wheel for psutil

### DISCUSSION THREAD

**Discussion by noob6t5** [termux/termux-app#4216] [Q&A]
### Problem description

When trying to install `psutil` in termux and tool dependent upon it . I am getting error's 
like 
   1.  psutil could not be installed from sources
    2. This error originates from a subprocess, and is not a problem with pip ,etc

You can check this too , (https://github.com/xnl-h4ck3r/waymore/issues/51)

### Steps to reproduce the behavior.

Run 
`pip install psutils`

### System information

* Termux application version: 1.42.1
* Android OS version: 11
* Device model: Redmi Y9

**Comment by 0xc18** [comment]
I think you misspelled. It's `psutil` not `psutils`.
Try running 
 `pip install psutil`

**Comment by noob6t5** [comment]
I am sorry i mistyped above , I did that too . But it's not working , Due to it other tool's like waymore isn't installing too

**Comment by 0xc18** [comment]
I can't help without knowing what I'm dealing with. Can you you send a screenshot of error message after running `pip install psutil`

**Comment by noob6t5** [comment]
Please check this once, I have described all here .

https://github.com/xnl-h4ck3r/waymore/issues/51

**Comment by 2096779623** [comment]
Try `apt upgrade -y`,it should be fixed,I can install it normally.

**Reply by noob6t5** [reply]
which version of termux are you using?

I did everything it isn't installed

**Reply by noob6t5** [reply]
It worked.
after pkg upgrade -y 
thanks 
![Uploading Screenshot_2024-10-19-18-58-10-051_com.termux.jpg…]()

**Comment by XIAYM-gh** [comment]
It seems that this thread is in the Announcements category. And it appears in my dashboard since I starred termux-app repository, which is confusing.

From my point-of-view, it should be moved into other categories.

Sorry for off-topic.

**Reply by twaik** [reply]
Done
